src/Concerns/ParallelCommand.php

```diff
@@ -48,15 +48,17 @@ protected function forkProcess(mixed ...$args): int
 
         if ($pid === -1) {
             return -1;
-        } elseif ($pid) {
+        }
+
+        if ($pid) {
             // Parent
             return $pid;
-        } else {
-            // Child
-            DB::reconnect();
-
-            exit($this->childHandle(...$args) ? 0 : 1);
         }
+
+        // Child
+        DB::reconnect();
+
+        exit($this->childHandle(...$args) ? 0 : 1);
```

Comment:
When forking I prefer to handle all cases in a single if-elseif-else block rather than basically early returns of varying indentation, think it reads a bit better to have all cases formatted the same.

Reply:
[`fabc09a` (#1278)](https://github.com/archtechx/tenancy/pull/1278/commits/fabc09a16bf32fca9a42bb04df042a516de5d520)